Smarty/templates/com_vtiger_workflow/EditTask.tpl

```diff
@@ -28,84 +28,191 @@
+									</div>
+								</div>
+							</div>
+							<!-- Created/Modified Time -->
+							<div class="slds-col slds-size_1-of-3 slds-text-align_center">
+								<div class="slds-form-element">
+									<div class="slds-form-element__control">
+										<div class="slds-select_container">
+											<select class="slds-select" name="select_date_field">
+												{foreach key=name item=label from=$dateFields}
+													<option value='{$name}' {if isset($trigger.field) && $trigger.field eq $name}selected{/if}>
+														{$label}
+													</option>
+												{/foreach}
+											</select>
+										</div>
+									</div>
+								</div>
+							</div>
+						</div>
+					</div>
 				</div>
-				<div class="slds-col slds-size_1-of-3">
-				<input type="checkbox" name="reevaluate" id="reevaluate" {if isset($task->reevaluate) && $task->reevaluate eq 1}checked{/if}>&nbsp;{$MOD.LBL_REEVALCONDITIONS}
+			</div>
+
+			<!-- Evaluate conditions on delayed execution -->
+			<div class="slds-col slds-size_1-of-3 slds-text-align_center slds-p-around_small">
+					<div class="slds-form-element">
+						<div class="slds-form-element__control">
+							<div class="slds-checkbox">
+								<input type="checkbox" name="reevaluate" id="reevaluate" {if isset($task->reevaluate) && $task->reevaluate eq 1}checked{/if}>
+								<label class="slds-checkbox__label" for="reevaluate">
+									<span class="slds-checkbox_faux"></span>
+									<span class="slds-form-element__label"> {$MOD.LBL_REEVALCONDITIONS} </span>
+								</label>
+							</div>
+						</div>
+					</div>			
+			</div>
+
+		</div>
+
+		<!-- Conditions -->
+		<div class="slds-grid slds-gutters">
+			<div class="slds-col">
+				{include file='com_vtiger_workflow/ListConditions.tpl' RecordSetTab=0}
+			</div>
+		</div>
+		
+		<div class="slds-page-header">
+			<div class="slds-grid slds-gutters">
+				<div class="slds-col slds-size_1-of-1">
+					<h1>
+						<span class="slds-page-header__title slds-truncate" title="{$MOD.LBL_TASK_OPERATIONS}">
+							<span class="slds-tabs__left-icon">
+								<span class="slds-icon_container" title="{$MOD.LBL_TASK_OPERATIONS}">
+									<svg class="slds-icon slds-icon_small" style="color:green;" aria-hidden="true">
+										<use xlink:href="include/LD/assets/icons/utility-sprite/svg/symbols.svg#task"></use>
+									</svg>
+								</span>
+							</span>
+							{$MOD.LBL_TASK_OPERATIONS}
+
+						</span>
+					</h1>
 				</div>
 			</div>
-			</td>
-		</tr>
-		</table>
-
-		<table border="0" cellpadding="5" cellspacing="0" width="100%" class="small">
-		<tr>
-			<td width='100%' nowrap="nowrap">
-			{include file='com_vtiger_workflow/ListConditions.tpl' RecordSetTab=0}
-			</td>
-		</tr>
-		</table>
-
-		<table class="tableHeading" border="0"  width="100%" cellspacing="0" cellpadding="5">
-			<tr>
-				<td class="big" nowrap="nowrap">
-					<strong>{$MOD.LBL_TASK_OPERATIONS}</strong>
-				</td>
-			</tr>
-		</table>
+		</div>
+
```

Comment:
please eliminate one of these two empty lines

Reply:
Ok